src/main/java/org/jitsi/videobridge/DtlsTransport.java

```diff
@@ -131,7 +131,7 @@ public DtlsTransport(Endpoint endpoint, boolean controlling, Logger parentLogger
                         getClass().getSimpleName() + "-outgoing-packet-queue",
                         TaskPools.IO_POOL,
                         this::handleOutgoingPacket,
-                        1024);
+                        DtlsTransportConfig.Config.queueSize());
```

Comment:
nit: I think in other places we've imported `DtlsTransportConfig.Config` so we can just refer to the config values via `Config.queueSize`